### PR TITLE
Update VS2015 build instructions.

### PR DESCRIPTION
**Pull request by ximinez**
* Include Python and Scons, and new OpenSSL lib path.

Reviewers: @vinniefalco, @miguelportilla, @seelabs, anybody else who builds with VS.

### CODE REVIEW

File: Builds/VisualStudio2015/README.md [hunk 1]
```diff
@@ -33,6 +29,9 @@ page, run the installer, and follow the directions.
 The **Visual Studio 2015 Community** edition is available for free, while paid editions
 may be used for an free initial trial period.
 
+Any version of Visual Studio 2015 may be used, including the Visual Studio Community
+Edition which is available under a free license.
+
```
Comment (seelabs) [inline Builds/VisualStudio2015/README.md]: The C++ compiler is no longer installed by default in visual studio. We should remind people to choose "Custom" installation, and under "Programming Languages" select "Visual C++"

Comment (MarkusTeufelberger) [inline Builds/VisualStudio2015/README.md]: "which is available under a free license" - I highly doubt that. It is probably available _for free_, but not under a _free (software) license_.

**Comment by ximinez** [comment]
I forgot to mention during the initial submission that, in addition to comparing the changes made here, it may be valuable to compare to the [Visual Studio 2013 instructions](https://wiki.ripple.com/Visual_Studio_2013_Build_Instructions) on the public wiki.

File: Builds/VisualStudio2015/README.md (hunk 1 above)
Comment (ximinez) [inline Builds/VisualStudio2015/README.md]: I've updated to address both of your comments. Let me know if there's still room for improvement.

**Comment by seelabs** [comment]
:+1:

**Comment by ximinez** [comment]
References to 32-bit Windows have been removed.

**Comment by ximinez** [comment]
> 1.8.3.2 is quite old and can run into problems (like signing). I would bump this to 2.6.2

Interesting. I still have 1.9.5, and haven't had any problems, including with signing. Obviously, we want them to have the latest version, but I think of it as "if your version is older than 1.8.3.2, you're gonna need to upgrade."  It's the same idea behind specifying scons as 2.3.5, even thought the latest version is 2.4.0.

**Comment by miguelportilla** [comment]
Specified software required within this doc is the current latest and much newer (2+ years) than this git version. I can't see a reason other than requiring the latest.

**Comment by ximinez** [comment]
> I think we should recommend the latest stable version of each requirement, that succeeds in this environment, with every document update.

Yeah, that makes a lot more sense when you put it that way. I'll update.

**Pull request closed without merging**